.github/workflows/labeler.yml

```diff
@@ -15,10 +14,31 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-      - uses: github/issue-labeler@v2.6
-        # We continue-on-error because the action is buggy and tries to remove labels when they don't exist
-        continue-on-error: true
+      - name: Label pull request
+        uses: phated/issue-labeler@master
+        id: issue-labeler
         with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
           configuration-path: .github/labeler.yml
           enable-versioned-regex: 0
+          sync-labels: 1
```

Comment:
The new version of the action has `sync-labels` option which avoids our hacky workaround from before.